Loop/View Controllers/ChartsTableViewController.swift

```diff
@@ -168,8 +166,11 @@ class ChartsTableViewController: UITableViewController, UIGestureRecognizerDeleg
     // References to registered notification center observers
     var notificationObservers: [Any] = []
 
-    var active: Bool = false {
-        didSet {
+    var active: Bool {
+        get {
+            return UIApplication.shared.applicationState == .active
+        }
+        set {
             log.debug("[reloadData] for app change to active: %d", active)
             reloadData()
```

Comment:
reloadData() has a lot of assertions about various states, so this would not actually optimize anything.  Also, if we needed to pause something, it would probably make more sense to do it in reloadData() rather than putting in the didSet block.